includes/assets/web-stories-block/scss/common.scss

```diff
@@ -0,0 +1,122 @@
+/* Common block styles */
+.web-stories-list {
+  &.has-archive-link {
+    margin-top: 30px !important;
+  }
+}
+.web-stories-list__story-wrapper {
+  position: relative;
+  cursor: pointer;
+
+  .is-view-type-grid &,
+  .is-view-type-carousel & {
+    min-width: var(--ws-story-min-width);
+  }
+
+  // Curved corners by default.
+  .is-style-default.is-view-type-grid & {
+    border-radius: 8px;
+    overflow: hidden;
+  }
+}
+
+.web-stories-list__story-placeholder {
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &::before {
+    content: '';
+    display: block;
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    background: linear-gradient(
+      180deg,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(0, 0, 0, 0.8) 100%
+    );
+    top: 0;
+    left: 0;
+  }
+
+  &::after {
+    content: '';
+    display: block;
+    padding-bottom: var(--ws-aspect-ratio);
+  }
+}
+
+.story-content-overlay {
+  padding: 10px;
+  line-height: var(--ws-overlay-text-lh);
+
+  .is-view-type-carousel &,
+  .is-view-type-grid & {
+    position: absolute;
+    bottom: 0;
+    color: var(--ws-overlay-text-color);
+    z-index: 1;
+  }
+
+  html[amp] & {
+    white-space: normal;
+  }
+
+  > *:not(:last-child) {
+    margin-bottom: 10px;
+  }
+
+  &__title {
+    font-size: var(--ws-font-size-title);
+    font-weight: 700;
+  }
+
+  &__author,
+  &__date {
+    font-size: var(--ws-font-size-author-date);
+  }
+
+  &__title,
+  &__excerpt {
+    display: -webkit-box; /* stylelint-disable-line value-no-vendor-prefix */
+    -webkit-line-clamp: 2; /* Browser support: https://caniuse.com/?search=line-clamp */
+    -webkit-box-orient: vertical; /* stylelint-disable-line property-no-vendor-prefix */
+    overflow: hidden;
+  }
+
+  &__excerpt {
+    -webkit-line-clamp: 3;
```

Comment:
Doesn't this need a lint error?

Reply:
you mean to ignore the lint, it should but it doesn't throw any lint errors in scan. 🤔